packages/yourloops/pages/hcp/patients/cards.tsx

```diff
@@ -152,56 +163,47 @@ function PatientCard(props: PatientElementCardProps): JSX.Element {
   }, [paperRef, isPendingInvitation, authHook, medicalData, patient, teamHook]);
 
   if (isPendingInvitation) {
-    const email = _.get(patient, "emails[0]", patient.username);
-    return (
-      <Paper id={`patients-list-card-${email}`} className={classes.patientPaperCard} data-userid={email}>
-        <div className={classes.patientDivPendingIcon}>
-          <AccessTimeIcon id={`patients-list-card-${email}-pendingicon`}/>
-        </div>
-        <Link
-          color="textPrimary"
-          id={`patients-list-card-${email}-email-link`}
-          href={`mailto:${email}`}
-          target="_blank"
-          rel="noreferrer">
-          {email}
-        </Link>
-      </Paper>
-    );
+    return (<PendingPatientCard patient={patient} onClickRemoveIcon={onClickRemoveIcon} />);
   }
 
-  const onClickFlag = (e: React.MouseEvent): void => {
-    e.stopPropagation();
-    onFlagPatient(userId);
-    metrics.send("patient_selection", "flag_patient", isFlagged ? "un-flagged" : "flagged");
-  };
-  const handleShowPatientData = (/* e: React.MouseEvent */): void => {
-    onClickPatient(patient);
-    metrics.send("patient_selection", "select_patient", isFlagged ? "flagged" : "un-flagged");
-  };
-
   return (
-    <Paper id={`patients-list-card-${userId}`} className={classes.patientPaperCard} ref={paperRef} data-userid={userId}>
-      <IconButton
-        id={`patients-list-card-${userId}-flag-btn`}
-        className={classes.flagButton}
-        aria-label={t("aria-flag-patient")}
-        size="small"
-        onClick={onClickFlag}>
-        {isFlagged ? <FlagIcon id={`patients-list-card-${userId}-flagged-icon`} /> : <FlagOutlineIcon id={`patients-list-card-${userId}-un-flagged-icon`} />}
-      </IconButton>
-      <Typography id={`patients-list-card-${userId}-fullname`} component="span">
-        {fullName}
-      </Typography>
-      <IconButton
-        id={`patients-list-card-${userId}-show-btn`}
-        className={classes.showPatientButton}
-        size="small"
-        onClick={handleShowPatientData}>
-        {<ArrowForwardIcon />}
-      </IconButton>
+    <Paper
+      id={`patients-list-card-${userId}`}
```

Comment:
All the ids here should have the same template as used in the table & pending cards.
Be sure they are all the same.

See my example on the pending card element.

Reply:
these ids where already here, I don't know if some system tests were written with those. If changes are needed, I'll do it later when adding new system tests for this new feature